html-forms/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get" class="html_forms">
+
+      <div class="html_forms">
+        <label for="email">Email</label>
+      </div>
+
+      <div class="html_forms">
+        <input type="email" id="mail" name="user_email"/>
+      </div>
+
+      <div class="html_forms">
+        <label for="question">Why is coding cool?</label>
+      </div>
+
+      <div class="html_forms">
+        <select id="reason" name="user_opinion">
+          <option>Select an option</option>
+          <option>Because it's interesting</option>
+          <option>Because it's cool</option>
+          <option>Because it's powerful</option>
+        </select>
+      </div>
+
+      <div class="html_forms">
+        <label for="choose_language">Favorite Language (choose one)</label>
+      </div>
+
+      <input type="radio" name="Language" id="Language_1" value="HTML" />
+      <label for="Language_1">HTML</label>
+      <input type="radio" name="Language" id="Language_2" value="CSS" />
+      <label for="Language_2">CSS</label>
+      <input type="radio" name="Language" id="Language_3" value="JavaScript" />
+      <label for="Language_3">JavaScript</label>
+
+      <div class="html_forms">
+        Would you like to sign up for more information?
+      </div>
+
+      <div class="html_forms">
+        <input type="checkbox" id="sign" name="signup">
+        <label for="scales">Sign Me Up!</label>
```

Comment:
I think you meant for this `for="sign"` to be associated with the checkbox?

Reply:
Oh yes I was looking at the example from MDN and I went on autopilot, thank you for the correction!